src/lib/isPassportNumber.js

```diff
@@ -57,6 +57,7 @@ const passportRegexByCountryCode = {
   TR: /^[A-Z]\d{8}$/, // TURKEY
   UA: /^[A-Z]{2}\d{6}$/, // UKRAINE
   US: /^\d{9}$/, // UNITED STATES
+  NG: /^([a-z]|[A-Z]\d{8})$/, // Nigeria  (One Alphabet followed by 8-digits)
```

Comment:
Same here, please add it before `NL`...

Reply:
@marvelefe please update your pull request :smile: 

thanks you @profnandaa